lib/dashlib.py

```diff
@@ -151,12 +160,20 @@ def create_superblock(proposals, event_block_height, budget_max, sb_epoch_time):
             )
         )
 
-        # else add proposal and keep track of total budget allocation
-        budget_allocated += proposal.payment_amount
-
         payment = {'address': proposal.payment_address,
                    'amount': "{0:.8f}".format(proposal.payment_amount),
                    'proposal': "{}".format(proposal.object_hash)}
+
+        add_size = 2 * (len(payment['address']) + len(payment['amount']) + len(payment['proposal']) + 3)
```

Comment:
Instead of just coding in numbers like '2' and '3' here, let's at least add a comment explaining that those are and why. But I'm not sure about digging into the serialization internals b/c then the serialize methods aren't a black box.